test/exit/kill.js

```diff
@@ -179,7 +179,7 @@ test('Cannot call .kill(null)', testInvalidKillArgument, null);
 test('Cannot call .kill(0n)', testInvalidKillArgument, 0n);
 test('Cannot call .kill(true)', testInvalidKillArgument, true);
 test('Cannot call .kill(errorObject)', testInvalidKillArgument, {name: '', message: '', stack: ''});
-test('Cannot call .kill([error])', testInvalidKillArgument, [new Error('test')]);
```

Comment:
GitHub action shows a weird `[error]` highlighted in red when a CI task prints the word `[error]` apparently. :)